mne/channels/channels.py

```diff
@@ -1335,3 +1335,63 @@ def _get_ch_info(info):
     return (has_vv_mag, has_vv_grad, is_old_vv, has_4D_mag, ctf_other_types,
             has_CTF_grad, n_kit_grads, has_any_meg, has_eeg_coils,
             has_eeg_coils_and_meg, has_eeg_coils_only)
+
+
+def create_1020_montage_mapping(ch_names, midline="z12"):
+    r"""Return dict mapping from ROI names to lists of picks for 10/20 setups.
+
+    This passes through all channel names, and uses a simple heuristic to
+    separate channel names into three Regions of Interest: Left, Midline and
+    Right. The heuristic is that channels ending on any of the characters in
+    `midline` are filed under that heading, otherwise those ending in uneven
+    numbers to the left, those in even numbers to the right. Other channels
+    are ignored. This is appropriate for 10/20 files, but not for other channel
+    naming conventions.
+    If an info object is provided, lists are sorted from posterior to anterior.
+
+    Parameters
+    ----------
+    ch_names : instance of info | list
+        Object a list of channel names will be extracted from. The picks will
+        be in relation to the position in this list. If an info object, lists
+        will be sorted by y value position of the channel locations, i.e.,
+        from front to back.
+    midline : str
+        Names ending in any of these characters are stored under the `midline`
+        key. Defaults to 'z12'. Note that capitalization is ignored.
+
+    Returns
+    -------
+    rois : dict
+        A dictionary mapping from ROI names to lists of picks (untegers).
+    """
+    if isinstance(ch_names, Info):
+        from .layout import find_layout
+        layout = find_layout(ch_names)
+        pos = layout.pos
+        ch_names = layout.names
+    else:
+        _validate_type(ch_names, (tuple, list, np.ndarray), "ch_names",
+                       "Info, or a collection of channel names.")
+        for ch_name in ch_names:
+            _validate_type(ch_name, "str")
+        pos = False
+
+    rois = dict(Midline=[], Left=[], Right=[])
```

Comment:
I prefer lower-case keys.

Reply:
These will probably end up printed as axis titles. Still prefer lower-case?

Reply:
As keys yes, as axis titles not. What about using `.title()` for the axis titles?

Reply:
Sure, that would work, but that's an extra call and extra code.

Reply:
Then just leave it as is.